source/develop/release-management/features/network/lldp.html.md

```diff
@@ -0,0 +1,461 @@
+    - description: Gathering LLDP information is enabled on the NIC
+      name: enabled
+      type: string
+      datatype: boolean
+
+    - description: Structured information of the TLV
+      name: properties
+      type: 
+      - *Tlv
+
+  Tlv: &Tlv
+    added: '4.2'
+    description: LLDP Information 
+    name: Tlv
+    properties:
+    - description: Human readable string to describe what the value is about.
+      name: name
+      type: string
+
+    - description: Structured information of the TLV
+      name: properties
+      type: *StringMap
+
+    - description: TLV type of IEEE 802.1AB.
+      name: type
+      type: string
+      datatype: uint
+
+    - description: organizationally unique identifier
+      name: oui
+      type: string
+      datatype: uint
+
+    - description: organizationally defined subtype
+      name: subtype
+      type: string
+      datatype: uint
+
+Host.getLldp:
+  added: '4.2'
+  description: Get lldp information of a NIC
+  params:
+    defaultvalue: no-default
+    description: Device name to query for lldp information
+    name: devicename
+    type: string
+  return:
+    description: Lldp information of a NIC
+    type: Lldp
+```
+
+
+## Entity Description
+
+The initial LLDP information is a list of type, length, value (TLV) tuples.
+To express multiple values, e.g. to list the allowed VLANs on a switch port,
+the TLV is sent multiple times.
+The value may contain structured data.
+This way the LLDP data is non-uniform.
+This non-uniform LLDP data will be mapped to the following uniform entities.
+
+The entity `Tlv` represents the TLV received physically by the oVirt host's NIC.
+A `Tlv` is associated to a `VdsNetworkInterface`, called
+[`HostNic` in REST-API](http://ovirt.github.io/ovirt-engine-api-model/4.2/#types/host_nic),
+has a `name`, a `type`, a list of `properties` and optional an `oui` and a
+`subtype`.
+
+![Class diagramm of Tlv](../../../../images/wiki/EntityTlv.png  "Class diagramm of Tlv")
+
+The `name` is a human readable string to describe what the value is about and
+may not be unique. The name is redundant, because it could be created from
+`type` and the optional `oui` and `subtype`. The purpose of `name` is
+to simplify the reading of the TLV.
+The `name` of a property is exactly the same string which is used in
+[IEEE 802.1AB][1] chapter 8.
+
+The `type` is the TLV type of [IEEE 802.1AB][1].
+
+Organizationally Specific `Tlv`s have the `type` of `0x7f` and the attributes
+`oui` and `subtype`.
+The `oui` holds the organizationally unique identifier, and the `subtype` the
+organizationally defined subtype.
+
+The `properties` are used to represent structured information transported by the
+TLV.
+
+The entity `lldp` has a list of all `Tlv`s of an interface and shows, if the
+gathering of the LLDP information is `enabled` on the interface. If it is 
+not `enabled`, there is a problem gathering the lldp information on the host.
+
+## CRUD
+
+For REST-API only not required.
```

Comment:
I don't understand your statement.
If you are advocating a DB-less implementation of this feature, please say so clearly.